SECURITY.WHITEPAPER.md

```diff
@@ -0,0 +1,153 @@
+# Datasafe Security Whitepaper
+
+Keystore
+ 
+*  [BCFKS](#Keystore-encryption) - to store keystores (AES256_KWP, PRF_SHA512, PBEWithHmacSHA256AndAES_256)
+*  [UBER](#Keystore-encryption) - to cache keys in memory (PBEWithSHAAnd3-KeyTripleDES-CBC)
+
+Path encryption
+
+*  [AES-GCM-SIV](#File-location-encryption)
+
+CMS Encryption
+
+*  [AES256_GCM, CHACHA20_POLY1305, SHA256withRSA](#Private-files-encryption-algoritm)
+
+## General information
+Datasafe is a flexible encryption library. It uses different encryption algorithms. Some of them can be 
```

Comment:
All of them not `some` can be configured. Most of them using configuration class/yaml file and the rest by Dagger/OverrridesRegistry